development/contributing.md

```diff
@@ -1,22 +1,46 @@
 ## Contributing
 
-### Git Commit Messages
+### Coding Best Practices
+
+#### Linting
+
+All code repositories adopt our standard linting rules found in the [flowforge/.github repository](https://github.com/flowforge/.github/blob/main/.eslintrc).
+
+We use [StandardJS](https://standardjs.com/), with one exception - 4 spaces not 2.
+
+If you're using VSCode, then we recommend using the [ESLint extenstion](https://github.com/Microsoft/vscode-eslint) and setting `all` for the `Eslint › Code Actions On Save: Mode` setting:
+
+<img width="429" alt="Screenshot 2022-04-07 at 10 11 04" src="https://user-images.githubusercontent.com/99246719/162164540-53cdf921-83a5-4d6e-9062-1552db24f453.png">
```

Comment:
We don't want to rely on external hosting of images here. They should be in the repo itself.